.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+How to write a good issue ? Read [this](https://github.com/containous/traefik/blob/master/.github/how-to-write-a-good-issue.md) and the [contributing guide](https://github.com/containous/traefik/blob/master/.github/CONTRIBUTING.md).
```

Comment:
We should use [relative links](https://github.com/blog/1395-relative-links-in-markup-files) so that reference don't break if we ever change the repo/hoster.